lib/browser/api/menu-item-roles.ts

```diff
@@ -143,7 +143,10 @@ export const roleList: Record<RoleId, Role> = {
     label: 'Toggle Developer Tools',
     accelerator: isMac ? 'Alt+Command+I' : 'Ctrl+Shift+I',
     nonNativeMacOSRole: true,
-    windowMethod: w => w.webContents.toggleDevTools()
+    webContentsMethod: wc => {
```

Comment:
can we test this? We should be able to trigger the role and then check `isDevToolsOpened()` i'd imagine?